templates/patient/partials/testing.html.twig

```diff
@@ -0,0 +1,2 @@
+{# THIS IS FOR POC ONLY - REMOVE BEFORE SUBMITTING PR #}
+{{ var1 }}
```

Comment:
hi @robertdown , looks like this should not of been included in the code that went into codebase?